pkg/virtctl/console/console.go

```diff
@@ -37,7 +37,7 @@ import (
 
 func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "console (vm)",
+		Use:     "console VM",
```

Comment:
To me, it seemed more obvious that vm is an argument the previous way this was documented.

Reply:
Hm, to use the same structure like kubectl, it would have to be `console VM_NAME`.